modules/gke-hub/variables.tf

```diff
@@ -0,0 +1,74 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+variable "features" {
+  description = "GKE hub features to enable."
+  type = object({
+    configmanagement    = bool
+    mc_ingress          = bool
+    mc_servicediscovery = bool
+  })
+  default = {
+    configmanagement    = true
+    mc_ingress          = false
+    mc_servicediscovery = false
+  }
+}
```

Comment:
Worth adding `nullable = false` here? Otherwise we should guard against this whole variable being null in main.

Reply:
good catch